### PR TITLE
Add googletest@1.15.2

### PR DESCRIPTION
**Pull request by derekmauro**

### CODE REVIEW

**Comment by derekmauro** [comment]
Is the CI check written in a way that it does not handle `dev_dependency = True`?

**Comment by Wyverald** [comment]
Indeed -- BCR CI is supposed to spot check your module from the viewpoint of a user (aka dependent). They don't see your dev deps.

**Review by Wyverald — APPROVED** [review]

File: modules/googletest/1.15.2/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+matrix:
+  bazel:
+  - 7.x
```
Comment (Mrzosef) [inline modules/googletest/1.15.2/presubmit.yml:3]: -

File: modules/googletest/metadata.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "1.13.0",
     "1.14.0",
     "1.14.0.bcr.1",
-    "1.15.0"
```
Comment (Mrzosef) [inline modules/googletest/metadata.json:19]: -

Comment (Mrzosef) [reply modules/googletest/metadata.json:19]: _

**Review by Mrzosef — CHANGES_REQUESTED** [review]

**Pull request merged**